src/main/java/com/yahoo/bullet/querying/evaluators/BinaryOperations.java

```diff
@@ -134,6 +135,22 @@ static TypedObject div(Evaluator left, Evaluator right, BulletRecord record) {
         });
     }
 
+    static TypedObject mod(Evaluator left, Evaluator right, BulletRecord record) {
+        return checkNull(left, right, record, (leftValue, rightValue) -> {
+            Type type = getArithmeticResultType(leftValue.getType(), rightValue.getType());
+            switch (type) {
+                case DOUBLE:
+                    return new TypedObject(Type.DOUBLE, getDouble(leftValue) % getDouble(rightValue));
```

Comment:
So modulus is Java is actually a remainder (works differently for negative numbers). floorMod works like modulus. Which were we planning to target with this op?

Reply:
I was actually going to call it remainder originally. But what's the usual behavior for %?

Reply:
Ok, looked through it more. Read up on how % is called modulo even though it's not actually modulo.

% behavior is what we want. Well, the operator is also called % so yea.